src/elasticsearch/user-elasticsearch.service.ts

```
@@ -0,0 +1,719 @@
+      console.error('Failed to initialize Elasticsearch index:', error);
+      throw new Error(
+        `Failed to initialize Elasticsearch index: ${error.message}`
+      );
+    }
+  }
+
+  async createUser(user: IUser) {
+    try {
+      // Handle empty date values
+      if (user.profile.dob === '') {
+        delete user.profile.dob;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)